bemserver_ui/templates/pages/timeseries/data/explore.html

```diff
@@ -58,7 +58,21 @@
                     <div is="app-ts-bucket-width" id="bucket" title="Duration" unit="day" value="1"></div>
                 </div>
             </div>
-            <div class="d-flex justify-content-end gap-2">
+            <div class="accordion app-sidebar-accordion rounded bg-primary bg-opacity-25" id="accordionTSExplore">
```

Comment:
Here you copied an accordion component from the sidebar. This is not really appropriate as this uses specific sidebar CSS style (`app-sidebar-accordion`, `app-sidebar-heading`...).
Instead I suggest you to mimic the "filter" accordion that is in list pages (users list, timeseries list, ...).